go.mod

```diff
@@ -67,7 +67,7 @@ require (
 	github.com/klauspost/cpuid v1.2.1 // indirect
 	github.com/klauspost/pgzip v1.2.1
 	github.com/knative/build v0.3.1-0.20190330033454-38ace00371c7
-	github.com/knative/pkg v0.0.0-20190330034653-916205998db9
+	github.com/knative/pkg v0.0.0-20190330034653-916205998db9 // indirect
```

Comment:
This is still there because of the dependency on `knative.dev/build` (which is deprecated and read-only now, **but did not migrate** to using `knative.dev`)